enclave-manager/server/server.go

```diff
@@ -217,32 +217,63 @@ func (c *WebServer) ListFilesArtifactNamesAndUuids(ctx context.Context, req *con
 	return resp, nil
 }
 
-func (c *WebServer) RunStarlarkPackage(ctx context.Context, req *connect.Request[kurtosis_enclave_manager_api_bindings.RunStarlarkPackageRequest], str *connect.ServerStream[kurtosis_core_rpc_api_bindings.StarlarkRunResponseLine]) error {
+func (c *WebServer) RunStarlarkPackage(ctx context.Context, req *connect.Request[kurtosis_enclave_manager_api_bindings.RunStarlarkPackageRequest], responseStream *connect.ServerStream[kurtosis_core_rpc_api_bindings.StarlarkRunResponseLine]) error {
 	apiContainerServiceClient, err := c.createAPICClient(req.Msg.ApicIpAddress, req.Msg.ApicPort)
-	runPackageArgs := req.Msg.RunStarlarkPackageArgs
-	shouldClonePackage := true
-	runPackageArgs.ClonePackage = &shouldClonePackage
 	if err != nil {
```

Comment:
For some reason the err-checking wasn't right after the function call; fixed this